openfisca_france/scripts/merge_ipp_tax_and_benefit_tables_with_parameters.py

```diff
@@ -65,21 +72,23 @@ def iter_ipp_values(node):
 def main():
     parser = argparse.ArgumentParser()
     parser.add_argument('-i', '--ipp-translations',
-        default = os.path.join(parameters_dir, 'ipp-tax-and-benefit-tables-to-parameters.yaml'),
```

Comment:
Where should we put such a documentation ?
it is mainly scripts to ease the regular merge between the baremes-ipp and the param.xml